NickvisionTubeConverter.GNOME/Views/AddDownloadDialog.cs

```diff
@@ -173,8 +173,16 @@ public AddDownloadDialog(AddDownloadDialogController controller, Gtk.Window pare
     private async Task ValidateAsync()
     {
         _spinnerVideoUrl.Start();
+        _rowFileType.SetSensitive(false);
+        _rowSavePath.SetSensitive(false);
+        _rowQuality.SetSensitive(false);
+        _rowSubtitle.SetSensitive(false);
         var checkStatus = await _controller.UpdateDownloadAsync(_rowVideoUrl.GetText(), (MediaFileType)_rowFileType.GetSelected(), _rowSavePath.GetText(), (Quality)_rowQuality.GetSelected(), (Subtitle)_rowSubtitle.GetSelected());
         _spinnerVideoUrl.Stop();
+        _rowFileType.SetSensitive(true);
+        _rowSavePath.SetSensitive(true);
+        _rowQuality.SetSensitive(true);
+        _rowSubtitle.SetSensitive(true);
```

Comment:
This should be
```csharp
_rowSubtitle.SetSensitive(((MediaFileType)_rowFileType.GetSelected()).GetIsVideo());
```

Reply:
Line 82 can then be removed as well.

Reply:
Line 82 is
```
_rowFileType.SetModel(Gtk.StringList.New(new string[] { "MP4", "WEBM", "MP3", "OPUS", "FLAC", "WAV" }));
```

why remove?

Reply:
Oh sorry I got the nunber wrong. Is should be whatever the line is that duplicates that same line I provided in the OnNotify event handler of rowSubtitle.